src/proctored-exam-settings/ProctoredExamSettings.test.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import ProctoredExamSettings from './ProctoredExamSettings';
+
+const defaultProps = {
+  courseId: 'course-v1%3AedX%2BDemoX%2BDemo_Course',
+};
+
+describe('ProctoredExamSettings', () => {
+  it('updates settings based on proctoring provider', () => {
+    const component = mount(<ProctoredExamSettings {...defaultProps} />);
```

Comment:
Can we get away with mounting shallowly? My goal in suggesting is to make this test of the top-level component less brittle to changes made in lower-level components in the future.

Reply:
I tried doing a shallow mount, but the data that I need to test are in paragon components, so it can't be shallow (from my understanding)